pages/api/verify.ts

```diff
@@ -44,22 +47,29 @@ const handler = async (req: NextApiRequest, res: NextApiResponse<Data>) => {
     });
     return;
   }
+  const discordConfigs = await DiscordServerConfigRepository.findBy({
+    discordServerId: discordMember.discordServerId,
+  });
 
   const messageHexHash = typedData.getMessageHash(messageToSign, body.account);
   const signatureVerified = await verifySignature(
     body.account,
     messageHexHash,
     body.signature,
-    discordMember.discordServer.starknetNetwork
+    body.network
   );
   if (!signatureVerified) {
     return res.status(400).json({ message: "Signature is invalid" });
   } else {
     discordMember.starknetWalletAddress = body.account;
-    await DiscordMemberRepository.save(discordMember);
+
     res.status(200).json({ message: "Successfully verified" });
     // Let's refresh its status immediatly
-    refreshDiscordMember(discordMember.discordServer, discordMember);
+    DiscordMemberRepository.save(discordMember);
+
+    for (let discordConfig of discordConfigs) {
+      refreshDiscordMember(discordConfig, discordMember, body.network);
```

Comment:
`refreshDiscordMember` does not take a network as 3rd parameter but a starkyModule